### PR TITLE
fix(FEC-11920): Skip Button Behavior

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

solves: FEC-11920

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/skip/skip.js [hunk 1]
```diff
@@ -36,7 +36,11 @@ class Skip extends Component {
   render(): React$Element<any> | void {
     if (this.props.playerSize !== PLAYER_SIZE.TINY && !this.props.loading) {
       return (
-        <div tabIndex="0" aria-label={this.props.label} className={skipStyle.btnSkip} onClick={this.props.onClick}>
+        <div
+          onClick={this.props.onClick}
+          className={[skipStyle.btnSkip, this.props.area === 'InteractiveArea' ? skipStyle.inTimeout : skipStyle.outTimeout].join(' ')}
```
Comment (yairans) [inline src/components/skip/skip.js:41]: inTimeout/outTimeout is not so meaningful. maybe something like bottomBarComponent.

File: src/components/skip/skip.scss [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+  padding: 7px;
+
+  &.interactive-area-position {
+    bottom: 0;
+  }
+
+  &.bottom-bar-position {
     top: 0;
-    right: 0;
     transform: translateY(-90%);
```
Comment (yairans) [inline src/components/skip/skip.scss:21]: 90% is correct anyway?
if you hover the ui and the bottom bar is shown while the timeout, and once the timeout is done the element moved to the bottom bar, do you see any movement, or it's in the exact place? in any player size?

File: src/components/skip/skip.js [hunk 3]
```diff
@@ -36,7 +36,14 @@ class Skip extends Component {
...
       return (
-        <div tabIndex="0" aria-label={this.props.label} className={skipStyle.btnSkip} onClick={this.props.onClick}>
+        <div
+          onClick={this.props.onClick}
+          className={[
+            skipStyle.btnSkip,
+            this.props.parentComponent === 'InteractiveArea' ? skipStyle.interactiveAreaPosition : skipStyle.bottomBarPosition
+          ].join(' ')}
+          tabIndex="0"
+          aria-label={this.props.label}>
```
Comment (yairans) [inline src/components/skip/skip.js:46]: use <Text

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/components/skip/skip.scss (hunk 2 above)
Comment (JonathanTGold) [reply src/components/skip/skip.scss:21]: Right is    -100% 
fixed

**Review by yairans — APPROVED** [review]

**Comment by yairans** [comment]
@JonathanTGold update the PR desc

**Pull request merged**